src/Components/test/testassets/TestServer/HostedInAspNetStartup.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace TestServer
+{
+    // Used for E2E tests that verify different overloads of MapFallbackToClientSideBlazor.
+    public class HostedInAspNetStartup
```

Comment:
I don’t quite understand why it’s necessary to add an entirely new test app for this. Doesn’t the existing BasicTestApp provide what’s needed for hosting an E2E test for this? I would have thought you could have created an anchor tag that references any of the existing static assets in the existing project.

Reply:
Removed now.